include/deal.II/fe/fe_tools.h

```diff
@@ -777,6 +782,29 @@ namespace FETools
                     const DoFHandler<dim,spacedim> &dof2,
                     const ConstraintMatrix         &constraints,
                     OutVector                      &z2);
+
+  /**
+   * Same as above for external parallel VectorTypes
+   * on a parallel::distributed::Triangulation
+   */
+  template <int dim, class VectorType, int spacedim>
+  void extrapolate_parallel(const DoFHandler<dim,spacedim> &dof1,
```

Comment:
But can't you just use the same signature and internally do
```
  if (dofhandler is based on a parallel triangulation)
    return internal::extrapolate_parallel (...)

  // treat the non-parallel case as before
```
? This way the non-standard name is kept entirely internally.